usermods/Tubes/node.h

```diff
@@ -276,9 +276,12 @@ class LightNode {
 
     void broadcastMessage(NodeMessage *message, bool is_rebroadcast=false) {
         // Don't broadcast anything if this node isn't active.
+#ifdef NODE_DEBUGGING
+        Serial.printf("broadcastMessage() - %s %s\n", status_code(), statusTimer.ended() ? "True" : "False");
+#endif
 
         if (status != NODE_STATUS_STARTED) {
-            if (status == NODE_STATUS_RECEIVING && statusTimer.every(STATUS_CHECK_RATE)) {
+            if (status == NODE_STATUS_RECEIVING && statusTimer.ended()) {
```

Comment:
this timer is only used for startup now ( perhaps it should be renamed ).  We just need to check that it expired to "start" and enable sending

Reply:
yep, let's rename this timer then.
This change will cause it to take up to 3sec before starting the node, vs. I believe the old code was checking every 200ms (to try to begin as fast as possible)

Seems like maybe this is a race condition between a) NODE_STATUS_RECEIVING being set + (0-200ms) vs. b) whatever actually needs to be true to be able to start syncing.
Setting it to 3sec(minus ID) might fix that just because it's so far after (b)?